taca_ngi_pipeline/deliver/deliver_mosler.py

```diff
@@ -0,0 +1,292 @@
+        """ Checks the delivery status of all project samples
+
+            :params sampleentries: a list of sample entry dicts to use instead
+                of fetching from database
+            :returns: True if all samples in this project has been successfully
+                delivered, False otherwise
+            THIS HAS BEEN COPIED AND PASTED FROM deliver.py ProjectDeliverer class
+        """
+        sampleentries = sampleentries or db.project_sample_entries(db.dbcon(), self.projectid).get('samples', [])
+        return all([self.get_delivery_status(sentry) == 'DELIVERED' for sentry in sampleentries])
+    
+    def db_entry(self):
+        """ Fetch a database entry representing the instance's project
+            :returns: a json-formatted database entry
+            :raises taca_ngi_pipeline.utils.database.DatabaseError:
+                if an error occurred when communicating with the database
+            THIS HAS BEEN COPIED AND PASTED FROM deliver.py ProjectDeliverer class
+        """
+        return db.project_entry(db.dbcon(), self.projectid)
+    
+    def update_delivery_status(self, status="DELIVERED"):
+        """ Update the delivery_status field in the database to the supplied 
+            status for the project specified by this instance
+            :returns: the result from the underlying api call
+            :raises taca_ngi_pipeline.utils.database.DatabaseError:
+                if an error occurred when communicating with the database
+            THIS HAS BEEN COPIED AND PASTED FROM deliver.py ProjectDeliverer class
+        """
+        return db.update_project(db.dbcon(), self.projectid, delivery_status=status)
+
+    def deliver_project(self):
+        """ Deliver all samples in a project to mosler
+            
+            :returns: True if all samples were delivered successfully, False if
+                any sample was not properly delivered or ready to be delivered
+        """
+        try:
+            logger.info("Delivering {} to {}".format(
+                str(self), self.expand_path(self.moslerdeliverypath)))
+            if self.get_delivery_status() == 'DELIVERED' \
+                    and not self.force:
+                logger.info("{} has already been delivered".format(str(self)))
+                return True
+            # right now, don't catch any errors since we're assuming any thrown 
+            # errors needs to be handled by manual intervention
+            status = True
+            # Open sftp session with mosler, in this way multiple tranfer will be possible
+            try:
+                transport=paramiko.Transport(self.moslersftpserver)
+                password = getpass.getpass(prompt='Mosler Password for user {}:'.format(self.moslersftpserver_user))
+                transport.connect(username = self.moslersftpserver_user, password = password)
+            except Exception as e:
+                print 'Caught exception: {}: {}'.format(e.__class__, e)
```

Comment:
If this is a debug statement, I'd say to remove it. Otherwise use e.g. `logger.error("")` instead